src/commands/tslint-wrapper.ts

```diff
@@ -0,0 +1,25 @@
+import { iCommand } from './commands';
+import { spawn } from 'child_process';
+import * as fs from 'fs';
+import { red, italic } from 'colors';
+import { emoji } from 'node-emoji';
+import Versions from './versions';
+import { Utils } from '../utils/utils';
+
+export default class TsLintWrapper implements iCommand {
+  execute(commands: string[]) {
+    new Versions().execute();
+    Utils.log('Running Command:');
+    Utils.log('tslint ' + commands.join(' '), italic);
```

Comment:
Can we try and run the `tslint` installed in `node_modules` instead of the global install?

e.g. `./node_modules/.bin/tslint`